outputs.tf

```diff
@@ -15,7 +15,7 @@
 #
 
 output ca_cert_pem {
-  value     = "${tls_self_signed_cert.root.cert_pem}"
+  value     = "${tls_self_signed_cert.root.*.cert_pem}"
```

Comment:
Can you explain why we have the splat?

Reply:
This is necessary to prevent errors in later versions of terraform that happen because we're calling an output that might not exist due to `count = 0`. It was a warning on the plan that I missed the first time around but figured we should probably add this in.

Reply:
Interesting. TIL